examples/notebooks/intro_notebook.ipynb

```diff
@@ -0,0 +1,307 @@
+    "\n",
+    "\n",
+    "- **`replace_xxxx`** : replacing a resource object will update the resource by replacing the existing spec with the provided one. Ex **`replace_namespaced_pod`** and **`replace_namespaced_deployment`**, to update pods and deployments respectively, by creating new replacements of the entire object. This performs operations similar to **`kubectl rolling-update`**, **`kubectl apply`** and **`kubectl replace`**.\n",
+    "\n",
+    "\n",
+    "- **`delete_xxxx`** : delete a resource. This performs operations similar to **`kubectl delete`**.\n",
+    "\n",
+    "\n",
+    "For Futher information see the Documentation for API Endpoints section in https://github.com/kubernetes-incubator/client-python/blob/master/kubernetes/README.md"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "from kubernetes import client, config"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "### Load config from default location"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "config.load_kube_config()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "### Create API and object instances"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "api_instance = client.ExtensionsV1beta1Api()\n",
+    "dep = client.ExtensionsV1beta1Deployment()\n",
+    "spec = client.ExtensionsV1beta1DeploymentSpec()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "### Fill required object fields (apiVersion, kind, metadata and spec)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "name = \"my-busybox\"\n",
+    "dep.metadata = client.V1ObjectMeta(name=name)\n",
+    "\n",
+    "spec.template = client.V1PodTemplateSpec()\n",
+    "spec.template.metadata =  client.V1ObjectMeta(name=\"busybox\")\n",
+    "spec.template.metadata.labels = {\"app\":\"busybox\"}\n",
+    "spec.template.spec = client.V1PodSpec()\n",
+    "dep.spec = spec\n",
+    "\n",
+    "container=client.V1Container()\n",
+    "container.image=\"busybox:1.26.1\"\n",
```

Comment:
your pod will actually crash because busybox does not run a default command. Either you switch to something like an nginx image, or you add a command like `sleep 3600` to make the busybox container actually run. Otherwise people will see pod enter a crashloop backoff.